src/index.md

```diff
@@ -9,20 +9,107 @@ of [Julia programming language](https://julialang.org/) ecosystem.
+* [ModuleDocstrings.jl -- ](https://github.com/JuliaDocs/ModuleDocstrings.jl)
 
 ## Packages
 
 ```@eval
 import Main: PackageDefinition, package_table_markdown
 [
+PackageDefinition(
+    "ANSIColoredPrinters", "https://github.com/JuliaDocs/ANSIColoredPrinters.jl",
+    [
+        "README" => "https://github.com/JuliaDocs/ANSIColoredPrinters.jl",
+    ],
+    [
+        "https://codecov.io/gh/JuliaDocs/ANSIColoredPrinters.jl/branch/main/graph/badge.svg" => "https://codecov.io/gh/JuliaDocs/ANSIColoredPrinters.jl",
+    ]
+),
+PackageDefinition(
+    "Changelog", "https://github.com/JuliaDocs/Changelog.jl",
+    [
+        "README" => "https://github.com/JuliaDocs/Changelog.jl",
+    ],
+    [
+        "https://codecov.io/gh/JuliaDocs/Changelog.jl/branch/master/graph/badge.svg" => "https://codecov.io/gh/JuliaDocs/Changelog.jl",
+    ]
+),
+PackageDefinition(
+    "CodeEvaluation", "https://github.com/JuliaDocs/CodeEvaluation.jl",
+    [
+        "README" => "https://github.com/JuliaDocs/CodeEvaluation.jl",
+    ],
+    [
+        "https://codecov.io/gh/JuliaDocs/CodeEvaluation.jl/branch/main/graph/badge.svg" => "https://codecov.io/gh/JuliaDocs/CodeEvaluation.jl",
+    ]
+),
+PackageDefinition(
+    "DemoCards", "https://github.com/JuliaDocs/DemoCards.jl",
+    [
+        "README" => "https://github.com/JuliaDocs/DemoCards.jl",
+    ],
+    [
+        "https://codecov.io/gh/JuliaDocs/DemoCards.jl/branch/master/graph/badge.svg" => "https://codecov.io/gh/JuliaDocs/DemoCards.jl",
+    ]
+),
+PackageDefinition(
+    "DocInventories", "https://github.com/JuliaDocs/DocInventories.jl",
+    [
+        "stable" => "https://juliadocs.org/DocInventories.jl/stable/",
+        "dev" => "https://juliadocs.org/DocInventories.jl/dev/",
+    ],
+    [
+        "https://codecov.io/gh/JuliaDocs/DocInventories.jl/branch/master/graph/badge.svg" => "https://codecov.io/gh/JuliaDocs/DocInventories.jl",
+    ]
+),
+PackageDefinition(
+    "DocStringExtensions", "https://github.com/JuliaDocs/DocStringExtensions.jl",
+    [
+        "stable" => "https://DocStringExtensions.juliadocs.org/latest/stable/",
+        "dev" => "https://DocStringExtensions.juliadocs.org/latest/dev/",
+    ],
+    [
+        "https://codecov.io/gh/JuliaDocs/DocStringExtensions.jl/branch/master/graph/badge.svg" => "https://codecov.io/gh/JuliaDocs/DocStringExtensions.jl",
+    ]
+),
+PackageDefinition(
+    "DocumentationGenerator", "https://github.com/JuliaDocs/DocumentationGenerator.jl",
+    [
+        "README" => "https://github.com/JuliaDocs/DocumentationGenerator.jl",
+    ],
+    [
+        "https://codecov.io/gh/JuliaDocs/DocumentationGenerator.jl/branch/master/graph/badge.svg" => "https://codecov.io/gh/JuliaDocs/DocumentationGenerator.jl",
+    ]
+),
+PackageDefinition(
+    "DocumentationGeneratorRegistry", "https://github.com/JuliaDocs/DocumentationGeneratorRegistry",
+    [
+        "README" => "https://github.com/JuliaDocs/DocumentationGeneratorRegistry",
```

Comment:
This repo could be omitted from the list

Reply:
So either we archive that repo (then the script will ignore it), or I can add it to an explicit exclude list. Preferences?